### PR TITLE
Login signup page

### PR DESCRIPTION
**Pull request by rohit8686**

### CODE REVIEW

File: Pages/Login/login.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        <div class="flex pt-1">
+          <a href="/Pages/Signup/signup.html" class="link">
+            Create new Account
+          </a>
+          <span class="material-icons-outlined"> chevron_right </span>
+        </div>
+      </form>
+    </div>
+  </body>
+</html>
```
Comment (romabulani) [inline Pages/Login/login.html:72]: In future, you can follow this practice - add one extra line at the end of the file, so github wont give this error

**Review by romabulani — APPROVED** [review]
Clean code and good use of Component Library, good to merge

**Review by Sharmaa-Ansh — APPROVED** [review]
good use of utility classes.

File: Pages/Login/login.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  font-size: 2rem;
+}
+.input,
+.full-width {
+  width: 100%;
+}
+.text-info {
+  color: var(--info-color);
+}
+.p-2 {
```
Comment (ravikumar1002) [inline Pages/Login/login.css:16]: try to make file for utility classes

File: Pages/Login/login.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
+@import "https://spiralui.netlify.app/index.css";
+
+.nav {
+  box-shadow: 0 0 20px 1px rgba(0, 0, 0, 0.3);
```
Comment (ravikumar1002) [inline Pages/Login/login.css:4]: use color variable

**Review by ravikumar1002 — APPROVED** [review]

File: Pages/Login/login.html (hunk 1 above)
Comment (rohit8686) [reply Pages/Login/login.html:72]: yeah okay

File: Pages/Login/login.css (hunk 2 above)
Comment (rohit8686) [reply Pages/Login/login.css:16]: okay

**Pull request merged**